packages/mobile/src/helpers/image/pick-image/pick-image.helper.ts

```diff
@@ -28,6 +40,14 @@ export const pickImageCamera = async (): Promise<Asset | null> => {
   if (response.didCancel) {
     notification.info(i18n.t('common:errors.DID_CANCEL'));
   }
+  if (
+    response?.assets?.[0]?.type !== undefined &&
+    ALLOW_IMAGE_TYPES.indexOf(response?.assets?.[0]?.type)
+  ) {
+    notification.error(i18n.t('errors.WRONG_FORMAT'));
+
+    return null;
+  }
 
   return response?.assets?.[0] || null;
```

Comment:
This part is the same for both functions  
```
if (response.errorCode && response.errorMessage) {
    notification.error(response.errorMessage, response.errorCode);
  }
   if (response.didCancel) {
     notification.info(i18n.t('common:errors.DID_CANCEL'));
   }
   if (
     response?.assets?.[0]?.type !== undefined &&
     ALLOW_IMAGE_TYPES.indexOf(response?.assets?.[0]?.type)
   ) {
     notification.error(i18n.t('errors.WRONG_FORMAT'));

     return null;
   }

   return response?.assets?.[0] || null;
```
I suggest moving it to the separate function, f.e `processImageRespose`